ci/tasks/test-gcs-blobstore-client-integration.sh

```diff
@@ -28,7 +28,7 @@ function clean_up {
 echo $google_json_key_data > key.json
 gcloud auth activate-service-account --key-file=key.json
 
-export GCS_SERVICE_ACCOUNT_KEY=`pwd`/key.json
+export GCS_SERVICE_ACCOUNT_KEY=$google_json_key_data
```

Comment:
The old version would populate the environment variable with a path to the key while the new version populates with the actual key.

This follows the change in convention in bosh-gcscli's testing and avoid unnecessary filesystem access.